install.py

```diff
@@ -16,7 +16,7 @@ def main():
         print()
         print ('packaging trapheus for use ...')
         try:
-            subprocess.run(package_command, shell=False, check=True)
+            subprocess.run(package_command, shell=True, check=True)
```

Comment:
This opens us up for a security injection attack ? Can we check this https://docs.python.org/3/library/subprocess.html#replacing-shell-pipeline